client/src/locales/translations/es_ES.json

```diff
@@ -0,0 +1,766 @@
+      "other": "Otra ayuda",
+      "otherDesc": "Necesito ayuda para conseguir comida/medicina/etc.",
+      "editProfile": "Editar Perfil",
+      "intro": "Presentación individual",
+      "website": "Página Web Personal",
+      "myActivity": "Mi Actividad",
+      "userActivity": "Actividad del Usuario",
+      "wantHelp": "Quiero ayudar",
+      "needHelp": "Necesito ayuda",
+      "firstNameRequired": "Se requiere el primer nombre",
+      "lastNameRequired": "Se requiere el apellido"
+    },
+    "org": {
+      "title": "Crear el Perfil de la Organización",
+      "editOrgProfile": "Editar el Perfil de la Organización",
+      "name": "Nombre de la Organización",
+      "email": "Correo Electrónico de la Organización",
+      "invalidEmail": "El correo electrónico no es válido",
+      "globalOrg": "Somos una organización global",
+      "seeking": "¿Qué estás buscando?",
+      "volunteers": "Voluntarios",
+      "donations": "Donaciones",
+      "staff": "Miembro de la organización",
+      "other": "Otros",
+      "typeAndIndustry": "Tipo y Ramo",
+      "type": "Tipo",
+      "types": [
+        "Compañía Startup",
+        "Empresa",
+        "Comunidad",
+        "Gobierno",
+        "Sector No Lucrativo",
+        "Universidad",
+        "Proveedor de Servicios de Salud",
+        "Otros"
+      ],
+      "industry": "Industria",
+      "industries": [
+        "Automóvil",
+        "Arquitectura",
+        "Artes y Artesanías",
+        "Sector Bancario",
+        "Biotecnología",
+        "Provisiones y Equipos",
+        "Organización Cívica y Social",
+        "Acontecimientos y Servicios Recreativos",
+        "Recaudación de Fondos",
+        "Relaciones Gubernamentales",
+        "Salud, Bienestar y Estado Físico",
+        "Salud Hospitalaria y Cuidado de Salud",
+        "Industrias",
+        "Servicios de Tecnología e Información",
+        "Servicios Jurídicos",
+        "Marketing y Publicidad",
+        "Instrumentos Médicos",
+        "Organización sin fines de lucro",
+        "Otros",
+        "Filantropía",
+        "Educación Primaria/Secundaria",
+        "Formación y Coaching Profesional",
+        "Investigación",
+        "Grupo de Expertos",
+        "Traducción y Localización"
+      ],
+      "typeIndustryError": "Selecciona el tipo de organización y la industria del menú desplegable",
+      "desc": "Descripción de la Organización",
+      "lang": "Idioma de la Organización",
+      "completeOrgProfile": "Completar el Perfil de la Organización",
+      "about": "Acerca de",
+      "activity": "Actividad",
+      "editAccount": "Editar Información de la Cuenta",
+      "updateSuccess": "La organización se actualizó con éxito",
+      "orgNameRequired": "Se requiere el nombre de la organización"
+    }
+  },
+  "auth": {
+    "signIn": "Iniciar sesión",
+    "joinNow": "Únete ahora",
+    "email": "Correo Electrónico",
+    "enterEmail": "Introduzca la contraseña",
```

Comment:
Oh, here is that error that I pointed out. Change this to "Introduzca el correo electrónico"